anthos-bm-utils/abm-nettest/nettest.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     spec:
       containers:
       - name: echoserver
-        image: gcr.io/google_containers/echoserver:1.4
+        image: gcr.io/anthos-baremetal-release/simplehttpserver:v0.3.1__linux_amd64
```

Comment:
We're changing the image name from echoserver to simplehttpserver? How do we go from 1.4 to v0.3.1?

Reply:
It's actually a completely different binary. We've introduced `simplehttpserver` under `nettest` repo, which makes `nettest` and `simplehttpserver` to share the same version number.